src/views/linear_layout.rs

```diff
@@ -145,6 +145,11 @@ impl LinearLayout {
         self.invalidate();
     }
 
+    /// Returns index of focused inner view
+    pub fn get_focus_index(&self) -> &usize {
```

Comment:
or `usize`?

Reply:
For types with small size like primitives, it's preferable to return by value since it's often cheaper to copy them than to provide a pointer indirection.

Also, an immutable reference to `LinearLayout`'s field may interfere with the need to use mutable references to `LinearLayout` or to move it altogether according to borrowing rules. If that's what you want to keep the value synchronized with the state of a `LinearLayout` without having to write the synchronization code, then that's fine. In most other cases it's better to return `usize`.

Reply:
thanks for clarification.
Updated